### PR TITLE
Feat: add the user doc of `vela top`

### PR DESCRIPTION
**Pull request by Hanmengnan**
### Description of your changes

add the user doc of `vela top` and the i18 doc 

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] Update `sidebar.js` if adding a new page.
- [x] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -131,6 +131,11 @@ module.exports = {
             'end-user/policies/shared-resource',
             'end-user/policies/apply-once',
             'end-user/policies/gc',
+            'how-to/dashboard/config/helm-repo',
+            'how-to/dashboard/config/image-registry',
+            'tutorials/access-application',
+            'tutorials/cloud-shell',
+            'tutorials/vela-top',
```
Comment (wonderflow) [inline sidebars.js]: vela top should be in `Day-2 Operations` instead of `CD Policies`.

File: sidebars.js [hunk 2]
```diff
@@ -131,6 +131,11 @@ module.exports = {
             'end-user/policies/shared-resource',
             'end-user/policies/apply-once',
             'end-user/policies/gc',
+            'how-to/dashboard/config/helm-repo',
+            'how-to/dashboard/config/image-registry',
+            'tutorials/access-application',
+            'tutorials/cloud-shell',
```
Comment (wonderflow) [inline sidebars.js]:
```suggestion
```

Comment (Hanmengnan) [reply sidebars.js]: OK, I have fixed it.

File: sidebars.js (hunk 1 above)
Comment (Hanmengnan) [reply sidebars.js]: OK, I have fixed it.

**Review by wonderflow — APPROVED** [review]
great job

**Pull request merged**